### ISSUE TITLE
for the baseline method

### ISSUE THREAD

**Issue by luoyi1hao** [cwmok/Fast-Symmetric-Diffeomorphic-Image-Registration-with-Convolutional-Neural-Networks#4]
Thank you for your previous help. I found some differences between the published code on Github and the paper. If I use this work as a baseline (comparison) method, should I follow the results directly implemented on Github, or follow the results of the code modified on Github by the paper?

**Comment by cwmok** [comment]
Thanks for your interest in our work. I think the changes on Github are minor. As such, please feel free to report any of these along with their hyperparameters. In addition, if you are more concerned with registration accuracy over diffeomorphism, please try to train SYMNet with the regularization parameter setting to 1 (i.e.: --smooth 1). It will significantly boost the registration accuracy in terms of the Dice coefficient, according to our experiments.

**Comment by luoyi1hao** [comment]
Thank you for your help.
